index.js

```diff
@@ -0,0 +1,12 @@
+import {
+  // eslint-disable-next-line no-unused-vars
+  inputBook, books, populateFields, Book,
+} from './modules/Books.js';
+
+import {
+  // eslint-disable-next-line no-unused-vars
+  contact, navList, navAdd, navContact, infoPage, addBookPage,
+} from './modules/navBar.js';
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Kindly remove the `eslint-disable` and fix the linters. The linters are there for a purpose of preventing bugs and they should not be avoided.